content/help/ops/traffic-management/deploy-guidelines/index.md

```diff
@@ -232,3 +232,49 @@ $ kubectl scale --replicas=0 deploy/istio-citadel -n istio-system
 {{< /text >}}
 
 This should stop Istio from restarting Envoy and disconnecting TCP connections.
+
+## Configure a multiple host gateway with SSL
+
+To configure the ingress gateway with TLS correctly for you to deploy multiple
+hosts, you must configure a unique port name with the following command:
```

Comment:
I don't think this covers the full extent of the issue. I think the general problem is that if you configure more than one server, either in a single Gateway (like below), or spread across more than one that use the same gateway implementation pods (e.g., `selector: istio: ingressgateway`) then you will encounter this problem. The command, below, is just one example.

Reply:
In this case, if there is more than one server with the same port number and the protocol is HTTPS, then the port names MUST be unique.

Reply:
Agreed

Reply:
Fair enough, but in that case you need to submit a PR against the release-1.0 branch, instead of master, and please clearly explain the general problem.

Reply:
I ve created #2332 to submit PR against release 1.0, my bad sry. Waiting to have your proposal regarding name generated for further release